commerce-headless-graphql/src/main/resources/com/liferay/commerce/headless/graphql/internal/servlet/commerce.graphqls

```diff
@@ -0,0 +1,15 @@
+type Query {
+	products: [Product]
+}
+
+
+type Product {
+	cpDefinitionId: String
+	title: String
+	facets: [Facet]
+}
+
+type Facet {
+	title: String
+	count: Int
+}
```

Comment:
Even though we probably don't have a pattern for `graphqls` files in Liferay, I don't think we have any (or at least very many) files that we allow to end in a newline -- so this file probably shouldn't either.